src/main/scala/learnshapeless/TaggedTypes.scala

```diff
@@ -33,26 +33,29 @@ object TaggedTypes {
   trait NameTag
   type Name = String @@ NameTag
   /** Tag the string "Katherine" as a Name*/
-  def ex_name: Name = ???
+  def ex_name: Name = tag2("Katherine").@@[NameTag]
```

Comment:
Perhaps also mention that `tag2` is not from Shapeless to the attendees - I'm sure people will be looking for it if you didn't tell them.